packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.spec.tsx

```diff
@@ -30,6 +30,7 @@ describe('UrlOptionsTable', function () {
 
   describe('with SearchParams', function () {
     beforeEach(function () {
+      cleanup();
```

Comment:
Why in the beforeEach and not afterEach?

Reply:
looks like a leftover